### DISCUSSION TITLE
Prevent window out of focus transparency

### DISCUSSION THREAD

**Discussion by alexhajdu** [microsoft/terminal#13682] [Ideas]
Hey 👋 
Not sure if not a windows limitation, but can we also keep transparency if the Terminal doesn't have focus? I tried to keep the Terminal always on top, but it was still rendered as a black background.

**Comment by kul-sudo** [comment]
Could you send a screenshot of this issue?

**Comment by zadjii-msft** [comment]
This is a feature request that's covered by #7158. If you set `useAcrylic: false`, then the "vintage transparency" effect will always be transparent.
